azext_hanaonazure/modules_sdk/models/hana_management_client_enums.py

```diff
@@ -48,3 +48,9 @@ class HanaInstancePowerStateEnum(str, Enum):
     stopped = "stopped"
     restarting = "restarting"
     unknown = "unknown"
+
+
+class HanaDatabaseContainersEnum(str, Enum):
```

Comment:
See above; this should be a boolean whether the specific HDB instance is MDC or not. The whole purpose is to provide downward support for (legacy) single-tenant DBs.

Reply:
This is auto-generated code. The reason that it is not a boolean is because the ARM team does not like to allow booleans in their configuration code. I thought that we had decided on this naming over a month ago when I was getting the swagger spec approved.